src/app/trainings-bonus/trainings-bonus.component.ts

```diff
@@ -25,14 +26,14 @@ export class TrainingsBonusComponent implements OnInit {
     }
 
     activeModal(trainingBonus = null) {
-        this.active = 'active';
+        this.active = true;
```

Comment:
ce `this.active` est useless, utilise le `@ViewChild('spectre-modal') modal` pour appeler le `this.modal.toggleModal` de `spectre-modal` directement